_pages/2024/shared-tasks/simultaneous.md

```diff
@@ -13,40 +13,172 @@ if you want to have a line break either put a blank line in between the text or
+- A **readme** file containing information needed to run the system, named `readme`.
+
+The submission should be compressed as a `tar.gz` file,
+and uploaded to the [dropbox](https://www.dropbox.com/scl/fo/g623o0jyndqahcblmz5gn/h?dl=0&rlkey=6e441oy6uhn64ldqxmbomn7c7). For data security reason, you might need to request the access before the uploading the file.
+
+The name of file should be
+
+```
+{team_name}_{target_modality}_{target_language}.tar.gz
+```
+
+For instance, if one submission is named `meta_text_de.tar.gz`, and `tar -xf meta_text_de.tar.gz` should have the following output
+
+```
+tar -xf meta_text_de_low.tar.gz -C meta_text_de_low
+> ls meta_text_de_low
+image.tar results readme
+```
+
+### Docker Image
+
+The docker image should already be built and able to run in a controlled environment, specifically an AWS [ap3.2xlarge](https://aws.amazon.com/ec2/instance-types/p3/) instance. Please make sure that the **docker image alone** is enough to run the system. All dependencies, such as checkpoints and dictionaries should be included inside the image.
+
+The follow command should be able to reproduce the same output as `results` in the submission:
+
+```bash
+# Load the image
+docker load -i my_docker_image.tgz
+
+# Start the service. Exposing the 2023 for evaluation
+docker run -p 2023:2023 --gpus all --shm-size 32G my_docker_image:latest
+
+# Evaluation
+simuleval --remote-eval --remote-port 2023 --source source.txt --target target.txt
+```
+
+The docker will start the service of the system, and `simuleval --remote-eval` will serve as a front-end for evaluation.
+
+### MuST-C v2.0 tst-COMMON
+
+The participants are required to submit the output of the system (the `--output` from `simuleval`) on MuST-C v2.0 tst-COMMON.
+The segmented data can be downloaded [here](https://dl.fbaipublicfiles.com/simultaneous_translation/iwslt2023/must-c_v2.0_tst-COMMON.tgz)
+The latency constraint will be check on the MuST-C v2.0 tst-COMMON set.
+
+### Readme
+
+A `readme` file should contains the following information
+
+- Instruction to run the system on each track and language direction.
+- Results on MuST-C v2.0 tst-COMMON of the default system.
+- Any other information needed to run the system
+
+### Example
+
+An example of training, evaluating and submission preparation with the baseline systems can be found [here](https://github.com/facebookresearch/fairseq/tree/iwslt2023/examples/simultaneous_translation)
 
 
 ## Evaluation
 
-<!-- Description of metrics used for evaluation, what the official ranking is based on, links to evaluation scripts -->
+The evaluation implementation will use the latest [SimulEval](https://github.com/facebookresearch/SimulEval) toolkit.
+We will use the new [remote evaluation](https://simuleval.readthedocs.io/en/latest/tutorials/remote_evaluation.html) mode.
+
+### Metrics
+The system's performance will be evaluated in two ways:
+
+- Translation quality metrics:
+  - BLEU
+  - BLASER
```

Comment:
For simplicity, maybe we can remove the BLASER this year.

Reply:
ok. for speech to speech, specify ASR-BLEU ?